Example/Kommunicate_ExampleUITests/KommunicateLoginAsVisitorUITests.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  KommunicateLoginAsVisitorUITests.swift
+//  Kommunicate_ExampleUITests
+//
+//  Created by Kommunicate on 02/07/20.
+//  Copyright © 2020 CocoaPods. All rights reserved.
+//
+
+import XCTest
+
+class KommunicateLoginAsVisitorUITests: XCTestCase {
+    enum GroupData {
+        static let typeText = "verifying Login as a visitor and FAQ button."
+        static let AppId = "TestAppId"
+    }
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        addUIInterruptionMonitor(withDescription: AppPermission.AlertMessage.accessNotificationInApplication) { (alerts) -> Bool in
+            if alerts.buttons[AppPermission.AlertButton.allow].exists {
+                alerts.buttons[AppPermission.AlertButton.allow].tap()
+            }
+            return true
+        }
+        let app = XCUIApplication()
+        if let appId = appIdFromEnvVars() {
+            app.launchArguments = ["-appId", appId]
+        }
+        app.launch()
+        sleep(5)
+        guard !XCUIApplication().scrollViews.otherElements.buttons[InAppButton.LaunchScreen.getStarted].exists else {
+            return
+        }
+    }
+    
+    func testFAQButton() {
+        let app = createConversation_Using_LoginAsVisitorButton()
+        let inputView = app.otherElements[AppScreen.chatBar].children(matching: .textView).matching(identifier: AppTextFeild.chatTextView).firstMatch
+        waitFor(object: inputView) { $0.exists }
+        inputView.tap()
+        inputView.typeText(GroupData.typeText) // typing message
+        app.buttons[InAppButton.ConversationScreen.send].tap()
+        let faqButton = app.navigationBars[AppScreen.kMConversationView]
+        waitFor(object: faqButton) { $0.exists }
+        faqButton.buttons[InAppButton.ConversationScreen.faqButton].tap()
```

Comment:
Here we have to verify if FAQ view has been loaded by checking the content, otherwise we are just tapping the button and coming back.

Reply:
Added check for FAQ web view